### PR TITLE
Adds Twilio integration for basic sending

### LINKED ISSUE #5: Connect our app to Twilio
In order to use Twilio we'll need to connect it to Laravel.

Twilio has documentation here: https://www.twilio.com/docs/sms/quickstart/php
And a Laravel/Twilio documentation: https://www.twilio.com/docs/sms/tutorials/server-notifications-php-laravel

After this issue our app should be able to send texts through twilio.

### PR DESCRIPTION
**Pull request by Chase-R**
Fixes #5 

Also adds basic logging, check out the new wiki page for details.

Required change to local setup! Check the wiki setup updates, step 3; you need the 3 new Twilio environment variables.

This PR also removes the '/vendor' directory from the repo.

### CODE REVIEW

File: app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+{
+    public function index() {
+        // Use format +18123334444 for the target phone number
+        $targetPhoneNumber = "+";
+        $targetMessage = "Check out this message!";
+        $this->_sendSms($targetPhoneNumber, $targetMessage);
+        return view('dashboard');
+    }
+
+    private function _sendSms($to, $message)
```
Comment (jetilling) [inline app/Http/Controllers/DashboardController.php]: Cant `$to` be `$targetPhoneNumber` or `$userPhoneNumber`? I feel like that would be a little more descriptive and readable.

File: app/Http/Controllers/DashboardController.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        try {
+            $client->messages->create(
+                $to,
+                [
+                    "body" => $message,
+                    "from" => $twilioNumber
+                ]
+            );
+            Log::info('Message sent to ' . $twilioNumber);
```
Comment (jetilling) [inline app/Http/Controllers/DashboardController.php]: This should be changed from 
```
Log::info('Message sent to ' . $twilioNumber);
```
to
```
Log::info('Message sent to ' . $targetPhoneNumber);
```
or whatever we call that variable

File: app/Http/Controllers/DashboardController.php (hunk 1 above)
Comment (Chase-R) [reply app/Http/Controllers/DashboardController.php]: Agreed.

File: app/Http/Controllers/DashboardController.php (hunk 2 above)
Comment (Chase-R) [reply app/Http/Controllers/DashboardController.php]: Nice catch.

File: app/Http/Controllers/DashboardController.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        Log::info("Attempting to send SMS to: " . $targetPhoneNumber . " with message: " . $message);
+        $accountSid = env('TWILIO_ACCOUNT_SID');
+        $authToken = env('TWILIO_AUTH_TOKEN');
+        $twilioNumber = env('TWILIO_NUMBER');
+
+        $client = new Client($accountSid, $authToken);
+
+        try {
+            $client->messages->create(
+                $to,
```
Comment (jetilling) [inline app/Http/Controllers/DashboardController.php]: This should be `$targetPhoneNumber` as well

**Pull request merged**